app/policies/manual_policy.rb

```diff
@@ -0,0 +1,11 @@
+class ManualPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  alias_method :new?, :index?
+  alias_method :create?, :index?
+  alias_method :edit?, :index?
+  alias_method :update?, :index?
+  alias_method :show?, :index?
```

Comment:
@rgarner I take your point about `:index?` having some kind of primary status, but I personally find the `alias_method` approach slightly easier to parse visually than the meta-programming approach (and the semantic equivalence doesn't offend me as much). This is all very subjective though.

I think this may be moot because we'll likely revise `ManualPolicy` to reflect the SPv1 rules.